gemma-core/src/main/java/ubic/gemma/core/analysis/preprocess/SplitExperimentServiceImpl.java

```diff
@@ -407,7 +403,7 @@ private Collection<FactorValue> cloneFactorValues( Collection<FactorValue> facto
         Collection<FactorValue> result = new HashSet<>();
         for ( FactorValue fv : factorValues ) {
             FactorValue clone = FactorValue.Factory.newInstance( ef );
-            clone.setCharacteristics( this.cloneCharacteristics( fv.getCharacteristics() ) );
+            clone.setCharacteristics( factorValueService.cloneCharacteristics( fv ) );
```

Comment:
It's preferable to do this so we can isolate the cloning logic in FactorValueDao and properly test it.